ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js

```diff
@@ -73,6 +142,88 @@ import { AggregationBehavior } from './nuxeo-aggregation-behavior.js';
     static get is() {
       return 'nuxeo-checkbox-aggregation';
     }
+
+    static get properties() {
+      return {
+        _buckets: {
+          type: Array,
+          value: () => [],
+        },
+        /**
+         * Make the element collapsible
+         */
+        collapsible: {
+          type: Boolean,
+          value: false,
+        },
+
+        /**
+         * Label of the element
+         */
+        label: {
+          type: String,
+          value: '',
+        },
+
+        _show: {
+          type: Boolean,
+          value: false,
+        },
+
+        _slicedBuckets: {
+          type: Array,
+          value: () => [],
+        },
+
+        _visibleBuckets: {
+          type: Array,
+          value: () => [],
+        },
+
+        /**
+         * If collapsible, the default visible items.
+         */
+        visibleItems: {
+          type: Number,
+          value: 8,
+        },
+      };
+    }
+
+    _checkCollapse(buckets) {
+      if (!this.collapsible && buckets.length) {
+        this._visibleBuckets = buckets;
+        return true;
+      }
+      return false;
+    }
+
+    _setBuckets(buckets) {
+      if (this.collapsible && buckets.length) {
+        this._buckets = buckets;
+        this._slicedBuckets = this._buckets.slice(0, this.visibleItems);
+        this._visibleBuckets = this._slicedBuckets;
+        return true;
+      }
+    }
+
+    _toggleShow() {
+      this._show = !this._show;
+      if (this._show) {
+        this._visibleBuckets = this._buckets;
+      }
+      if (!this._show) {
+        this._visibleBuckets = this._slicedBuckets;
```

Comment:
The only place where it could justify having the remaining items saved would be here, but I guess the performance impact wouldn't be that much. 🤷‍♂️